appserver/connectors/connectors-runtime/src/main/java/com/sun/enterprise/resource/deployer/AdministeredObjectDefinitionDeployer.java

```diff
@@ -336,6 +337,26 @@ public void setEnabled(String value) throws PropertyVetoException {
             return administeredObjectProperties;
         }
 
+        @Override
+        public Property addProperty(Property property) {
+            throw new UnsupportedOperationException();
```

Comment:
@lprimak could be, needs more investigation on the API. I followed the same pattern from resource enable/disable/redeploy methods.